### PR TITLE
Sort the columns in the Today's Lab Utilization table when clicking on table heads

### PR DESCRIPTION
**Pull request by Hexhu**
When a table head is clicked (like "% Busy" or "Desktop"), the column will be sorted in ascending order. Click again to sort in descending order. Hold the "Control" key while clicking to reset the column order to default.

A demo is available at [trojan.ocf.berkeley.edu:8794/stats/](http://trojan.ocf.berkeley.edu:8794/stats/). Scroll down and click on the "% Busy" header to see the effect.

### CODE REVIEW

**Comment by ja5087** [comment]
Thanks for the change!

Frontend for reference:

image

Can we add some UI elements to indicate that this is sortable? Right now it's not clear that it is, maybe something like 
image would work.

File: ocfweb/stats/templates/stats/summary.html [hunk 1]
```diff
@@ -48,25 +48,38 @@ <h4>{{printer}}</h4>
...
+                {% endfor %}
+            </tbody>
         </table>
     </div>
 {% endblock %}
+
+{% block inline_js %}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.31.3/js/jquery.tablesorter.min.js"></script>
+<script>
+$(document).ready(function() {
```
Comment (dkess) [inline ocfweb/stats/templates/stats/summary.html:81]: This line can be replace with just `$(function() {`

Comment (Hexhu) [reply ocfweb/stats/templates/stats/summary.html]: Shortened it. They are the same though. Plz review :3

**Comment by Hexhu** [comment]
TODO: add arrow images to table heads to indicate that the columns are sortable
- pick relevant `tablesorter-header` section from https://github.com/Mottie/tablesorter/blob/master/css/theme.default.css
- use arrows to indicate `headerSortUp`, `headerSortDown` and `sorter-false` states
- save style at `ocfweb/static/scss/pages/stats/_summary.scss`

**Comment by Hexhu** [comment]
Visual indicator added @ja5087 
Demo at http://trojan.ocf.berkeley.edu:8794/stats/

**Pull request merged**